yt_dlp/extractor/veoh.py

```diff
@@ -123,3 +130,91 @@ def _real_extract(self, url):
+            'info_dict': {
+                'id': 'PiensaLibre',
+                'title': 'PiensaLibre (Uploads)'
+            },
+            'playlist_mincount': 2
+        }]
+
+    _USERSINFO_ENDPOINT = 'https://www.veoh.com/users/published/videos'
+    _VIDEO_BASEURL = 'https://www.veoh.com/watch/'
+
+    def _get_authtoken(self, page, uploader=None):
+        webpage = self._download_webpage(page, uploader)
+        return self._search_regex(
+            r'csrfToken: "(?P<token>[0-9a-zA-Z]{40})"', webpage,
+            'request token', group='token')
+
+    def _get_videos(self, uploader, authtoken=None):
+        if authtoken is None:
+            authtoken = self._get_authtoken(f'https://www.veoh.com/users/{uploader}', uploader)
+        totalVids = 0
+        for i in itertools.count():
+            payload = json.dumps({'username': uploader, 'maxResults': 16, 'page': i + 1, 'requestName': 'userPage'}).encode('utf-8')
+            headers = {
+                'x-csrf-token': authtoken,
+                'content-type': 'application/json;charset=UTF-8'
+            }
+            for retry in self.RetryManager():
+                try:
+                    response = self._download_json(
+                        self._USERSINFO_ENDPOINT, uploader, data=payload, headers=headers,
+                        note=f'Downloading videos page {i + 1}')
+                    if not response['success']:
+                        raise ExtractorError('unsuccessful veoh user videos request')
+                    break
+                except ExtractorError as e:
+                    raise e
+
+            def resolve_entry(*candidates):
+                for cand in candidates:
+                    if not isinstance(cand, dict):
+                        continue
+                    permalink_url = url_or_none(self._VIDEO_BASEURL + cand['permalinkId'])
+                    if permalink_url:
+                        return self.url_result(
+                            permalink_url,
+                            VeohIE.ie_key() if VeohIE.suitable(permalink_url) else None,
+                            str_or_none(cand['permalinkId']), cand['title'])
+
+            for e in response['videos'] or []:
+                yield resolve_entry(e)
+
+            totalVids += len(response['videos'])
+            if (totalVids == response['totalRecords']) or (len(response['videos']) == 0):
+                break
+
+    def _extract_videos(self, uploader, playlist_title, playlist_id):
+        return {
+            '_type': 'playlist',
+            'id': playlist_id,
+            'title': playlist_title,
+            'entries': self._get_videos(uploader)
+        }
+
+    def _real_extract(self, url):
+        mobj = self._match_valid_url(url)
+        uploader = mobj.group('user')
+
+        return self._extract_videos(
+            uploader,
+            f'{str_or_none(uploader)} (Uploads)',
+            str_or_none(uploader))
```

Comment:
uploader will always be a string
```suggestion
           uploader)
```